index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
```

Comment:
`<h1>
			<span class="subTitle">A Viget Exploration</span>
			<img src="assets/title.svg" alt=" - Improving Medical Adherence">
		</h1>`
Fixed. I am considering naming the '<span>' as 'small-title' instead. What is an example of a class name that defines its content well?